internal/gke/logging.go

```diff
@@ -68,6 +82,10 @@ func (cl *CloudLoggingClient) Log(entry *protos.LogEntry) {
 	if entry.TimeMicros == 0 {
 		entry.TimeMicros = time.Now().UnixMicro()
 	}
+	if cl.filterLogEntry(entry) {
+		// Do not export the log entry if it doesn't satisfy the export filters.
+		return
+	}
```

Comment:
Doesn't really matter, but maybe we should move this `if` statement above the one before it so we don't call `time.Now().UnixMicro()` for logs we filter out.

Reply:
Good point. Done